atd-etl/app/process/helpers_request_download.py

```diff
@@ -0,0 +1,198 @@
+"""
+CRIS - Request Download Helper
+Author: Austin Transportation Department, Data and Technology Office
```

Comment:
`Author: Austin Transportation Department, Data and Technology Services`

Reply:
Will refactor that :/